macros/grants/grants.yml

```diff
@@ -0,0 +1,75 @@
+version: 2
+
+macros:
+  - name: grant_database_ownership_access
+    description: This macro grants ownership privilege to the specific roles within the database
+    docs:
+      show: true
+    arguments:
+      - name: rolename
+        type: string
+        description: Name of the role eg- ANALYST
+
+  - name: grant_database_usage_access
+    description: This macro grants database usages privilege to the specific roles within the database. To get access on the database, the role needs to have usage privilege in the database.
```

Comment:
specific roles? or specified rolename

Reply:
This macro grants usage privilages to the specified rolename inside the database. To get access on the database, the rolename needs to have usage privilege in the database.

Reply:
done